icechunk-python/python/icechunk/store.py

```diff
@@ -12,6 +18,20 @@
     from icechunk import Session
 
 
+def _byte_request_to_tuple(
+    byte_request: ByteRequest | None,
+) -> tuple[int | None, int | None]:
+    if byte_request is None:
```

Comment:
Use the match, Luke.

Reply:
The problem is that these are dataclasses that i dont think i can send to rust? IDK

Reply:
oh you can match in pythonjnow lol

Reply:
python has match!